pages/common/clifm.md

```diff
@@ -3,34 +3,34 @@
 > The command line file manager.
 > More information: <https://github.com/leo-arch/clifm>.
 
-- Change to/open the file whose ELN (entry list number) is 12 (ELN can be replaced by the actual file name, e.g. /etc):
+- Start CliFM:
+
+`clifm`
+
+- Open the file or directory whose ELN (entry list number) is 12:
 
 `12`
 
 - Create a new file and a new directory:
 
 `n file dir/`
 
-- Go back and forth in the directory history list:
+- Search for PDF files in the current directory:
 
-`Shift-Left (or Alt-j) / Shift-Right (or Alt-k)`
+`/*.pdf`
 
 - Select all PNG files in the current directory:
 
 `s *.png`
 
-- Search for PDF files in the current directory:
-
-`/*.pdf`
-
-- Remove the file whose ELN is 24 (use t to trash the file instead):
+- Remove the previously selected files (use `t` to send the files to the recycle bin instead):
 
-`r 24`
+`r sel`
 
-- Copy selected files into the current directory:
+- Learn more:
```

Comment:
Sorry to nitpick, but would this read better perhaps?
```suggestion
- Display detailed help:
```

Reply:
Looks good to me.